### PR TITLE
Allow selectURL() inside top level Fenced Frame

### PR DESCRIPTION
**Pull request by xyaoinum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,7 +346,9 @@ The privacy properties of shared storage are enforced through limited output. So
...
 
-selectURL() is disallowed in nested Fenced Frames. This is to prevent leaking lots of bits all at once via selectURL() chaining (i.e. a fenced frame can call selectURL() to add a few more bits to the fenced frame's current URL and render the result in a nested fenced frame). Though chaining seems quite useful, and we intend to revisit this. Use cases that will benefit from selectURL() being allowed from inside the top level fenced frame: [issue](https://github.com/WICG/fenced-frame/issues/44).
+selectURL() is disallowed in nested Fenced Frames. This is to prevent leaking lots of bits all at once via selectURL() chaining (i.e. a fenced frame can call selectURL() to add a few more bits to the fenced frame's current URL and render the result in a nested fenced frame). Use cases that will benefit from selectURL() being allowed from inside the top level fenced frame: [issue](https://github.com/WICG/fenced-frame/issues/44).
```
Comment (jkarlin) [inline README.md:349]: I don't quite understand what is being proposed here. Are you allowing selectURL() within selectURL fenced frames?

Comment (xyaoinum) [reply README.md:349]: Yes. selectURL() within top level selectURL FFs are allowed. By "nested FF" I intended to mean a FF tree with depths >= 2.

Comment (jkarlin) [reply README.md:349]: This is very subtle. Perhaps make it super clear what the limitation is? Something like:

selectURL() can be called in a top-level fenced frame, but not from within a nested fenced frame.

Comment (xyaoinum) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -346,10 +346,12 @@ The privacy properties of shared storage are enforced through limited output. So
...
 
-selectURL() is disallowed in Fenced Frame. This is to prevent leaking lots of bits all at once via selectURL() chaining (i.e. a fenced frame can call selectURL() to add a few more bits to the fenced frame's current URL and render the result in a nested fenced frame). Though chaining seems quite useful, and we intend to revisit this.
+selectURL() can be called in a top-level fenced frame, but not from within a nested fenced frame. This is to prevent leaking lots of bits all at once via selectURL() chaining (i.e. a fenced frame can call selectURL() to add a few more bits to the fenced frame's current URL and render the result in a nested fenced frame). Use cases that will benefit from selectURL() being allowed from inside the top level fenced frame: [issue](https://github.com/WICG/fenced-frame/issues/44).
+
+selectURL() is only availale in fenced frames that originate from shared storage (i.e. not available in [FLEDGE](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) generated fenced frames).
```
Comment (jkarlin) [inline README.md:351]: s/availale/available/

**Comment by jkarlin** [comment]
overall lgtm w/ nit

File: README.md (hunk 2 above)
Comment (xyaoinum) [reply README.md]: Done.

**Pull request merged**